Strings/1234/README.md

```diff
@@ -0,0 +1,93 @@
+# Problema:
+
+Uma sentença é chamada de dançante se sua primeira letra for maiúscula e cada letra subsequente for o oposto da letra anterior. Espaços devem ser ignorados ao determinar o case (minúsculo/maiúsculo) de uma letra. Por exemplo, "A b Cd" é uma sentença dançante porque a primeira letra ('A') é maiúscula, a próxima letra ('b') é minúscula, a próxima letra ('C') é maiúscula, e a próxima letra ('d') é minúscula.
+
+**Problema completo:** https://www.urionlinejudge.com.br/judge/pt/problems/view/1234
+
+# Resoluçāo:
+
+Neste exercício temos como entrada uma sentença, e devemos transforma-la em uma sentença dançante, isto é, uma sentença em que há uma alternação entre letras maiúsculas e minúsculas, sendo que a primeira letra desta frase deve ser maiúscula. Desta forma, para ter essa alternancia, as posições ímpares terão letras maiúsculas e as pares letras minúsculas.
+
+Primeiramente, vamos declarar a biblioteca `string.h` pois usaremos as funções `gets` e `strlen` .
+
+``` c
+    #include <string.h>
+```
+
+Então iremos declarar a cadeia de caracteres `sentenca` que irá armezar a frase que iremos fazer as modificações, ela tem tamanho 50 pois este é o tamanho máximo que uma sentença deste exercício pode ter. A seguinte declaramos os inteiros `i` e `count` , ambas variáveis contadoras, sendo que `i` é uma variável auxiliar dentro do `for` e `count` é usado para verificar o número da posição das letras da frase, se a posição que estar é par ou ímpar. Por último temos `tam` que guardará o tamanho da sentença.
```

Comment:
Então iremos declarar a cadeia de caracteres `sentenca` que irá armazenar a frase que iremos fazer as modificações. Ela tem tamanho 50, pois é o tamanho máximo que uma sentença deste exercício pode ter. A seguir, declaramos os inteiros `i` e `count` , ambas variáveis contadoras, sendo que `i` é uma variável auxiliar dentro do `for` e `count` é usado para verificar o número da posição das letras da frase, se a posição que está é par ou ímpar. Por último temos `tam`, que guardará o tamanho da sentença.